package.json

```diff
@@ -29,8 +29,8 @@
   },
   "homepage": "https://github.com/automation-stack/electron-sudo#readme",
   "scripts": {
-    "build": "node_modules/webpack/bin/webpack.js --config ./webpack/config.babel.js",
-    "build-win32": "node node_modules\\webpack\\bin\\webpack.js --config webpack\\config.babel.js",
+    "build": "webpack --config webpack\\config.babel.js",
```

Comment:
Oh, no %) We can't do it, we must use local webpack from node_modules and system-specific slash. Please revert to original version.